src/views/scans/__tests__/viewScansList.test.tsx

```diff
@@ -0,0 +1,166 @@
+  let asFragment;
+  const mockedDownloadReport = jest.fn();
+
+  beforeEach(async () => {
+    // we don't care about properly rendering timestamps in this test; mocking this
+    // helper makes our mock data simpler
+    jest.spyOn(helpers, 'getTimeDisplayHowLongAgo').mockImplementation(() => moment().from(moment()));
+    // we don't need to actually download anything either, so let's mock the downloader
+    // (this is a bit trickier because downloadReport is nested in another function)
+    const useScanApiModule = require('../../../hooks/useScanApi');
+    jest.spyOn(useScanApiModule, 'useDownloadReportApi').mockImplementation(() => {
+      return {
+        downloadReport: mockedDownloadReport
+      };
+    });
+    // mock axios to return dummyScanList
+    jest.spyOn(axios, 'get').mockResolvedValue({ data: { results: dummyScanList } });
+    // now let's render the ScanViewList...
+    const rendered = render(wrappedScanViewList());
+    asFragment = rendered.asFragment;
+    // ...wait until data is loaded
+    await waitFor(() => {
+      rendered.getByText('downloadable scan');
+    });
+  });
+  afterEach(() => {
+    jest.clearAllMocks();
+  });
+  it('sanity check rendered scanViewList', () => {
+    // ...ensure axios was called
+    expect(axios.get).toHaveBeenCalledTimes(1);
+    // ...and make sure it matches our snapshot
+    expect(asFragment()).toMatchSnapshot('scanViewList');
+  });
+
+  it('sanity check kebab menu', () => {
+    const { scanRow, kebabBtn } = clickKebabMenu(downloadableScanName);
+    expect(kebabBtn.getAttribute('aria-expanded')).toBe('true');
+    expect(scanRow).toMatchSnapshot('expanded kebab menu');
+    // clicking on it again should make it go away
+    act(() => {
+      fireEvent.click(kebabBtn);
+    });
+    expect(kebabBtn.getAttribute('aria-expanded')).toBe('false');
+    expect(scanRow).toMatchSnapshot('kebab menu hidden');
+  });
+
+  it('ensure only the first scan has a downloadable report', async () => {
+    // see dummyScanList; only the first scan should have a downloadable report
+    // grab the download button for the first scan
+    const { downloadBtn } = clickKebabMenu(downloadableScanName);
+    // the button should not be disabled
+    expect(downloadBtn.hasAttribute('disabled')).toBeFalsy();
+    // and clicking on it should...
+    act(() => {
+      fireEvent.click(downloadBtn);
+    });
+    // ...download the report
+    expect(mockedDownloadReport).toHaveBeenCalledWith(downloadableReportId);
+    mockedDownloadReport.mockReset();
+    // repeat the process for other scans, which should not be downloadable
+    const otherScans = [1, 2, 3, 4];
+    otherScans.forEach(n => {
+      const { downloadBtn: button } = clickKebabMenu(`not downloadable ${n}`);
+      expect(button.hasAttribute('disabled')).toBeTruthy();
+      act(() => {
+        fireEvent.click(downloadBtn);
+      });
+      expect(mockedDownloadReport).not.toHaveBeenCalled();
+    });
```

Comment:
@cdcabrera just to be sure we are on the same page, if I remove all the proposed tests but this one (this was the true goal of testing here), would you still be against this PR?